analysis/os_reports/dataset_definition_os_reports.py

```diff
@@ -99,4 +99,20 @@
     hospital_admissions.admission_method.is_in(["11", "12", "13"])
 ).where(
     hospital_admissions.admission_date.is_on_or_between(dod_ons - days(30), dod_ons)
-).count_for_patient()
\ No newline at end of file
+).count_for_patient()
+
+## Emergency admissions
+dataset.emadm_1m = hospital_admissions.where(
+    hospital_admissions.admission_method.is_in(['21', '2A', '22', '23', '24', '25', '2D'])
```

Comment:
Was it a deliberate choice not including "2B", "2C",  and "28"

Reply:
This was copied from the old study definition but I will double check of course when I'm back from AL.